src/sources/opentelemetry/mod.rs

```diff
@@ -66,45 +66,63 @@ pub struct OpentelemetryConfig {
 
 /// Configuration for the `opentelemetry` gRPC server.
 #[configurable_component]
+#[configurable(metadata(docs::examples = "Self::default()"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 struct GrpcConfig {
     /// The address to listen for connections on.
     ///
     /// It _must_ include a port.
+    #[configurable(metadata(docs::examples = "0.0.0.0:4317"))]
+    #[configurable(metadata(docs::examples = "localhost:4317"))]
     address: SocketAddr,
 
     #[configurable(derived)]
     #[serde(default)]
     tls: Option<TlsEnableableConfig>,
 }
 
+impl Default for GrpcConfig {
+    fn default() -> Self {
+        Self {
+            address: "0.0.0.0:4317".parse().unwrap(),
+            tls: None,
```

Comment:
cc @jszwedko @tobz what do you think?

Reply:
This is likely an outcome of how it's being serialized, since the JS code that builds the examples see the field, even if it's `null`, and that's what triggers it to be included.

You could try annotating the field, or any other optional fields, with `#[serde(skip_serializing_if = "Option::is_none")]`.

Reply:
Changed in https://github.com/vectordotdev/vector/pull/15703/commits/bb6b80250c093433494d0e25c37564c20a214733